standup/auth0/tests/test_views.py

```diff
@@ -113,4 +113,4 @@ def test_success(self, settings, db, client, messages_catcher):
             )
 
             # Assert we get redirected to the front page
-            assert is_redirect_to(resp, 'http://testserver/')
+            assert is_redirect_to(resp, '/')
```

Comment:
The HTTP 302 class thingy sets Location to just the path rather than the protocol, host, port and path like it did before. So I had to adjust all the checks accordingly.